spatialprofilingtoolbox/workflow/assets/cggnn.nf

```diff
@@ -85,29 +88,40 @@ process run_cggnn {
     path "${output_prefix}_model.pt"
 
 
-    shell:
-    '''
+    script:
+    """
+    #!/bin/bash
+
+    strata_option=\$( if [[ "${strata}" != "all" ]]; then echo "--strata ${strata}"; fi)
+    disable_channels_option=\$( if [[ "${disable_channels}" == "true" ]]; then echo "--disable_channels "; fi)
+    disable_phenotypes_option=\$( if [[ "${disable_phenotypes}" = "true" ]]; then echo "--disable_phenotypes"; fi)
+    target_name_option=\$( if [[ "${target_name}" != "none" ]]; then echo "--target_name '${target_name}'"; fi)
+    in_ram_option=\$( if [[ "${in_ram}" == "true" ]]; then echo "--in_ram"; fi)
+    merge_rois_option=\$( if [[ "${merge_rois}" == "true" ]]; then echo "--merge_rois"; fi)
+    prune_misclassified_option=\$( if [[ "${prune_misclassified}" == "true" ]]; then echo "--prune_misclassified"; fi)
+    upload_importances_option=\$( if [[ "${upload_importances}" == "true" ]]; then echo "--upload_importances"; fi)
```

Comment:
Do these strings no longer need spaces after them? If so, we should remove the space after `--disable_channels` for consistency.

Reply:
That is correct.
Typically in bash stuff, if whitespace is needed, it should be added at the last moment, since many intermediate steps are whitespace killers.